Assets/HoloToolkit/BuildAndDeploy/Editor/BuildDeployWindow.cs

```diff
@@ -453,6 +444,15 @@ private void AppxBuildGUI()
             }
 
             var curScriptingBackend = PlayerSettings.GetScriptingBackend(BuildTargetGroup.WSA);
+
+            if (curScriptingBackend == ScriptingImplementation.WinRTDotNET)
+            {
+                EditorGUILayout.HelpBox(".NET Scripting backend is depreciated, please use IL2CPP.", MessageType.Warning);
```

Comment:
When did this recommendation change? Also, is it specific to newer versions of Unity (better IL2CPP support)?

Reply:
Unity officially made .NET support depreciated: https://blogs.unity3d.com/2018/03/28/updated-scripting-runtime-in-unity-2018-1-what-does-the-future-hold/

Reply:
For now, can we wrap that in a 2018_1_OR_NEWER? Just to be safe

Reply:
Not really sure I agree with the 2018, maybe 2017.

In either case it's completely optional and devs don't have to switch it.

Reply:
Maybe just reword it to be "has been deprecated in Unity 2018, please consider using IL2CPP"?

Reply:
I'd like devs to submit tickets to Unity against IL2CPP in any version so that we get consistent across any editor version.

Reply:
Fair enough...

Reply:
Is "depreciated" here correct? I always thought it was "deprecated."

Reply:
Also, I'm not sure I read in there anywhere that .NET is not longer supported as a backend. All I read is that the .NET 3.5 Scripting Runtime Version is being removed in favor of .NET 4.x and .NET Standard 2.0. Am I missing something? There's no mention of IL2CPP or .NET as backends at all.

Reply:
https://docs.unity3d.com/2018.2/Documentation/Manual/windowsstore-dotnet.html

Reply:
We may want to hold off on pushing toward IL2CPP until vNext then. Do we really want to change the recommendation in master?

Reply:
Yup, just trying to be proactive.

Reply:
I strongly disagree.  We def want people to get used to the idea early and I'd like devs to submit tickets to Unity against IL2CPP in any version so that we get consistent across all editor versions.

Reply:
Besides it's not like we're _forcing_ them to use IL2CPP. It's just a heads up.

Reply:
Then we should say "please consider using" :)

Reply:
And let's not forget Kurtis noting the spelling error for "deprecated" :)